app/browser/api/ledger.js

```diff
@@ -1436,6 +1484,11 @@ const roundtrip = (params, options, callback) => {
     parts.pathname = parts.path
   }
 
+  if (options.windowP) {
+    roundTripFromWindow({url: urlFormat(parts), verboseP: options.verboseP}, callback)
```

Comment:
minor: `params.verboseP` is never used in roundTripFromWindow

Reply:
added some logging, so we use it now 😃